.github/workflows/edge.yml

```diff
@@ -46,6 +45,8 @@ jobs:
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
why do we do this? does it break otherwise ?

Reply:
yeah, the docs for GoReleaser say:

> Note the fetch-depth: 0 option on the Checkout workflow step. It is required for GoReleaser to work properly. Without that, GoReleaser might fail or behave incorrectly.

this is the description of the parameter:

>Number of commits to fetch. 0 indicates all history for all branches and tags.